cloudigrade/account/tests/test_helper.py

```diff
@@ -0,0 +1,128 @@
+"""Collection of tests for ``account.tests.helper`` module.
+
+Because even test helpers should be tested!
+"""
+import datetime
+import random
+import uuid
+
+from django.test import TestCase
+
+from account.models import Account, Instance, InstanceEvent
+from account.tests import helper
+from util.tests import helper as util_helper
+
+
+class AccountHelperTest(TestCase):
+    """Test helper functions test case."""
+
+    def test_generate_generate_account_default(self):
```

Comment:
is the double `generate` here and in fhte next test intentional?

Reply:
![](http://i0.kym-cdn.com/entries/icons/facebook/000/000/246/copypasta.jpg)